### DISCUSSION TITLE
API requests count

### DISCUSSION THREAD

**Discussion by seathwolf** [wp-privacy/wp-api-privacy#23] [General]
First thank you for making this plugin! 

I have a question about the count of API requests. I've been running this plugin on my test sites and I was really surprised how many hits I get. Yesterday I installed it on one of my sites, that is small local site nothing really special. This is what I got this morning.... 

**The number of API requests that have been modified since activation are: 951**

That is after 12 hours of activation! I didn't modified any setting.

Is that normal what you guys see? Because that is a lot of hits from WP in just 12 hours. 

Thanks.

**Comment by duanestorey** [comment]
It does seem high, but it depends what options you configured.  If you are adjusting entries for non WP API calls too, and you have some other plugins installed, then basically any external call will have an adjustment and it'll show up that data.  You can install the HTTP Requests Manager plugin if you want, and then it'll log all the HTTP requests and you can see them all.  Let me know how it goes.

**Reply by seathwolf** [reply]
I will do that and let you know thank you.
